_posts/2021-2-01-Workday-Overview.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Website Overview
+description: 
+summary: 
+tags: 
+---
+Today, I started work with Typing Club and made some progress. After Typing Club, I continue adding to my website. This time I added a search menu to my website and added functional links for the menu to work. I added the input on the main html page, and added the styling and functioning to style.css and script.js. I than created a pull request for Kajari to look over and leave moment on the changes to commit on the coding.
```

Comment:
Add a link to your website! And talk about the css file and the javascript file, what they do and how you pulled them in to your website. Also talk about linking to the other pages that you created last time!